<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Controller</class>
 <widget class="QWidget" name="Controller">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="windowIcon">
   <iconset resource="static.qrc">
    <normaloff>:/img/blackpiece.png</normaloff>:/img/blackpiece.png</iconset>
  </property>
  <widget class="Board" name="chessboard" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>750</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="panel" native="true">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>0</y>
     <width>250</width>
     <height>750</height>
    </rect>
   </property>
   <widget class="QPushButton" name="undoButton">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>550</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>悔棋</string>
    </property>
   </widget>
   <widget class="QWidget" name="blackPanel" native="true">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>50</y>
      <width>200</width>
      <height>250</height>
     </rect>
    </property>
    <widget class="QLabel" name="blackImage">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>64</y>
       <width>200</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>blackImage</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLCDNumber" name="blackTimer">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>125</y>
       <width>150</width>
       <height>50</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="blackLable2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>200</y>
       <width>200</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="whitePanel" native="true">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>250</y>
      <width>200</width>
      <height>250</height>
     </rect>
    </property>
    <widget class="QLabel" name="whiteImage">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>64</y>
       <width>200</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>whiteImage</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLCDNumber" name="whiteTimer">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>125</y>
       <width>150</width>
       <height>50</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="whiteLable2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>200</y>
       <width>200</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="returnButton">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>625</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Board</class>
   <extends>QWidget</extends>
   <header location="global">board.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="static.qrc"/>
 </resources>
 <connections/>
</ui>
